### PR TITLE
feat(flypad-settings): Move Simbrief Import section & introduce navigraph section 

### PR DESCRIPTION
**Pull request by alepouna**
## Summary

Moves the Simbrief Import description from ATSU/AOC to 3rd Party Settings, as it is being moved with https://github.com/flybywiresim/aircraft/pull/8548. 

This PR also introduces a section for Navigraph in the '3rd Party Settings' as it was missing. Please add reviews for the text, as I am not sure how correct it is in terms of grammar and sense. 

I am also attaching here my screenshots for replacement in settings.md Lines 223 & 265 but I am unsure if they are fine, LMK. 

FlightSimulator_nj5Hm5gSVd
FlightSimulator_TWiUTRO3iK

### Location

docs/fbw-a32nx/feature-guides/flypados3/settings.md 

https://[docs-git-fork-alepouna-aircraft-8548-flybywire.vercel.app/fbw-a32nx/feature-guides/flypados3/settings](https://docs-git-fork-alepouna-aircraft-8548-flybywire.vercel.app/fbw-a32nx/feature-guides/flypados3/settings)

Discord username (if different from GitHub): alepouna

### CODE REVIEW

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md [hunk 1]
```diff
@@ -234,6 +234,15 @@ Settings for integrations with various 3rd party applications
     <span class="imagesub">Click on the menu icons in this image to see other flyPad pages.</span>
 </div>
 
+#### Navigraph Integration 
+
+- Navigraph Account Link
+    - Wizard to connect your Navigraph Account with the flypad.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/settings.md]:
```suggestion
    - Setup wizard to connect your Navigraph Account with the flypad.
```

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md [hunk 2]
```diff
@@ -234,6 +234,15 @@ Settings for integrations with various 3rd party applications
...
+- Override SimBrief User ID
+    - Allows you use a custom Simbrief User ID if do not with to connect your Navigraph account or override the existing account link.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/settings.md]:
```suggestion
    - Allows users to input a custom Simbrief User ID. For more information see [A32NX simBrief Integration](../simbrief/#setup-a32nx-simbrief-integration).
```
You might need to check the link here. (Most of the information is already described on the page). The main issue is that we don't have a specific header for this as it's in an admonition. Could implement this if required within the scope of this PR.

Reference page:
[fbw-a32nx/feature-guides/simbrief/#setup-a32nx-simbrief-integration]

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md [hunk 3]
```diff
@@ -234,6 +234,15 @@ Settings for integrations with various 3rd party applications
...
 </div>
 
+#### Navigraph Integration 
+
+- Navigraph Account Link
+    - Wizard to connect your Navigraph Account with the flypad.
+- Override SimBrief User ID
+    - Allows you use a custom Simbrief User ID if do not with to connect your Navigraph account or override the existing account link. 
+- Automatically Import SimBrief Data
+    - Imports latest SimBrief flight automatically when starting the flyPad.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/settings.md:244]:
```suggestion
    - Imports the latest SimBrief flight automatically when powering on the flyPad.
```

**Review by Valastiri — DISMISSED** [review]
Should be prudent to double check the SimBrief integration page here https://docs.flybywiresim.com/fbw-a32nx/feature-guides/simbrief/#setup-a32nx-simbrief-integration

Since it's a dedicated page and the flypad is more of a brief overview of the features shown on a page.

Your images are fine just make sure to optimize them for the web. Thanks for this!

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md (hunk 2 above)
Comment (Sleinmaster) [reply docs/fbw-a32nx/feature-guides/flypados3/settings.md]:
```suggestion
    - Allows users to input a custom SimBrief User ID. For more information see [A32NX simBrief Integration](../simbrief/#setup-a32nx-simbrief-integration).
```
One additional correction based on Vals.

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md [hunk 4]
```diff
@@ -275,9 +284,7 @@ Settings for integration with various data and information sources.
     - Choose which weather data provider should be used (Aircraft only - does not change sim weather).
     - MeteoBlue is the weather service the sim uses as well.
 - TAF Source:
-    - Choose which Terminal Area Forecast (TAF) service should be used.
-- Automatically Import SimBrief Data
-    - Imports latest SimBrief flight automatically when starting the flyPad.   
+    - Choose which Terminal Area Forecast (TAF) service should be used.
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/feature-guides/flypados3/settings.md]:
```suggestion
    - Choose, which Terminal Area Forecast (TAF) service should be used.   
```

**Review by Sleinmaster — DISMISSED** [review]

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md (hunk 2 above)
Comment (Valastiri) [reply docs/fbw-a32nx/feature-guides/flypados3/settings.md]: Ah thanks. Hid my review on this part. Please use Slein's

**Comment by alepouna** [comment]
Review comments added, I will optimize the images and upload them. Thanks @Valastiri & @Sleinmaster

**Comment by Valastiri** [comment]
@2hwk I'm a little confused why a review is requested when the A32NX PR is on do not merge. Please advise here or feel free ping me in Discord :)

Edit: This PR is also missing images cc: @alepouna

**Comment by 2hwk** [comment]
> @2hwk I'm a little confused why a review is requested when the A32NX PR is on do not merge. Please advise here or feel free ping me in Discord :)
> 
> Edit: This PR is also missing images cc: @alepouna

I think I meant to hit dismiss review not re-request, my bad

**Comment by Valastiri** [comment]
Affirm thanks!

**Comment by alepouna** [comment]
> Edit: This PR is also missing images cc: @alepouna

Images updated to an optimized version of those found in the PR description.

**Pull request merged**